docs/source/requirements.rst

```diff
@@ -0,0 +1,10 @@
+requirements
+============
+
+## Minimum requirements
+
+To run the ci-framework locally on your machine you will need:
+
+* 32 GB RAM
```

Comment:
20GB

Reply:
If fact, with our current implementation based on a nested VM, I won't recommend anything less that 32GB, as the VM is configured to use 22GB and the direct deployment/non-nested VM is not yet there. Maybe we should rephrase that whole sentence.

Reply:
Oh, you are right, I was thinking about the VM having 20gb not the whole machine...

Reply:
@pablintino @cjeanner what option do you prefer? github page or readthedoc? 

with readthedoc permission is rewuired to integrate a webhook (https://readthedocs.org/dashboard/ci-framework/integrations/) so maybe github  page for the project is better.

Reply:
I have not strong preferences. As long as we can automate in a convenient, non-invasive way...

IIRC, ghpage requires pushing to a specific branch - not sure how we may automate this actually.

readthedoc is kind of "the" place to be - many projects are using it in a convenient way. Integrating the webhook isn't a blocker at this stage - especially since it's a well known, heavily used and therefore tested/audited hook.

Reply:
great. then we just need to fix access permissions. Who is the guy to ask for the change?